File: js/one.js
function freefall(val, isD){

        if(isD == true){
            const distance = parseFloat(val)
            var t = (2 * distance/9.81) **0.5
            var t = parseFloat(t)

            return t.toFixed(2)
        }
        else{
            const time = parseFloat(val)
            var d = 0.5 * 9.81 * time**2
            return d.toFixed(2)



        }
    }




console.log(freefall(0.45, false))

File: bhargavi/one.py
def freeFall(val,isD):
	val = float(val)
	while isD:
		t = (2*val/9.81) ** 0.5
		return round(t,2)
	else:
		d = 0.5 * 9.81 * val**2
		return round(d,2)






# print(freeFall(1,True))
# print(freeFall(0.45,False))